app/components/Progress.tsx

```diff
@@ -0,0 +1,16 @@
+interface ProgressProps {
+    answer: number | null;
+    index: number;
+    totalQuestion: number;
+  }
+export default function Progress ({totalQuestion, index, answer}: ProgressProps) {
+return (
+    <div className=" mb-12">
+         <progress
```

Comment:
I thought you were going to use [MUI's progress bar](https://mui.com/material-ui/react-progress/#linear-with-label)